base/Enums.jl

```diff
@@ -62,6 +94,8 @@ end
 Member values can be converted between the enum type and `BaseType`. `read` and `write`
 perform these conversions automatically.
 
+`Base.Enums.basetype` function returns `BaseType`.
```

Comment:
We probably want `[Base.Enums.basetype](@ref)` and also have the docstring for `basetype` show up in the list of docstrings in the online manual by adding it to the list here https://github.com/JuliaLang/julia/blob/master/doc/src/base/base.md#special-types.